src/icon/flexibilityIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export const FlexibilityIcon = (props: Icon) => (
+  <BaseIcon {...props} viewBox="0 0 24 24">
+    <g fill={props.iconColor} fillRule="evenodd">
```

Comment:
Do we really need the `g `since we have only one `path`?

Reply:
Thanks, indeed, I modified the SVG I received by removing a useless path but didn't see the now lonely <g>